e2e/cypress/integration/enterprise/guest_accounts/guest_identification_spec.js

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @guest_account
+
+/**
+ * Note: This test requires Enterprise license to be uploaded
+ */
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, getRandomId} from '../../../utils';
+
+const authenticator = require('authenticator');
+
+describe('Guest Accounts', () => {
+    let sysadmin;
+    let testTeam;
+    let testChannel;
+    let adminMFASecret;
+    const username = 'g' + getRandomId(); // usernames has to starts with a letter.
+
+    before(() => {
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+        });
+
+        cy.apiRequireLicenseForFeature('GuestAccounts');
+
+        // # Log in as a team admin.
+        cy.apiAdminLogin().then((res) => {
+            sysadmin = res.user;
+        });
+    });
+
+    after(() => {
+        // Login back as admin.
```

Comment:
Please add `#` or `*` whichever is applicable for consistency. Also in other places.
```suggestion
        // # Login back as admin.
```

Reply:
Sure, but some comments are not a step that are defined in the test script. Actually all of the comments without a mark is a test step. So, i think I can add `#` to all of them?

Reply:
Yes, please. Thanks!